checkov/sca_image/runner.py

```diff
@@ -162,20 +162,36 @@ def iterate_image_files(self, abs_fname: str, report: Report, runner_filter: Run
             if image_referencer.is_workflow_file(abs_fname):
                 images = image_referencer.get_images(file_path=abs_fname)
                 for image in images:
-                    image_report = self.get_image_report(dockerfile_path=abs_fname, image_id=image,
+                    image_report = self.get_image_report(dockerfile_path=abs_fname, image=image,
                                                          runner_filter=runner_filter)
                     merge_reports(report, image_report)
 
-    def get_image_report(self, dockerfile_path: str, image_id: str, runner_filter: RunnerFilter) -> Report:
+    def get_image_report(self, dockerfile_path: str, image: Image, runner_filter: RunnerFilter) -> Report:
         """
 
         :param dockerfile_path: path of a file that might contain a container image
-        :param image_id: sha of an image
+        :param image: Image object
         :param runner_filter:
         :return: vulnerability report
         """
         report = Report(self.check_type)
 
+        scan_result = self.scan(image.image_id, dockerfile_path, runner_filter)
+        if scan_result is None:
+            return report
+        self.raw_report = scan_result
+        result = scan_result.get('results', [{}])[0]
+        vulnerabilities = result.get("vulnerabilities") or []
+        self.parse_vulns_to_records(report, result, f"{dockerfile_path} ({image.name} lines:{image.start_line}-{image.end_line} ({image.image_id}))", runner_filter, vulnerabilities,
+                                    file_abs_path=os.path.abspath(dockerfile_path))
+        return report
+
+    def get_image_id_report(self, dockerfile_path: str, image_id: str, runner_filter: RunnerFilter) -> Report:
+        """
+        THIS METHOD ONLY EXISTS FOR BACKWARD COMPATABILITY WITH IMAGE THAT COMES FROM CLI PARAMETER
```

Comment:
@schosterbarak It's still the way we support scanning custom images
```suggestion
        THIS METHOD HANDLES CUSTOM IMAGE SCANNING THAT COMES DIRECTLY FROM CLI PARAMETERS
```